src/main/scala/firrtl/ir/IR.scala

```diff
@@ -360,6 +360,11 @@ case class Conditionally(
   def foreachString(f: String => Unit): Unit = Unit
   def foreachInfo(f: Info => Unit): Unit = f(info)
 }
+
+object Block {
+  def apply(head: Statement, tail: Statement*): Block = Block(head +: tail)
```

Comment:
Word of caution, varargs are implemented in Scala by WrappedArrays, so this is going to have to copy the entire array

Reply:
In response to my own point, this is only usable by static code (ie. you can't splat to it), so I doubt anyone is going to have large `tails` here so it probably doesn't matter in practice.

Reply:
The `Seq` factory that you avoid uses varargs, too :)